frontend/src/entry/model/mirroredModels/ReleasesSelector.tsx

```diff
@@ -0,0 +1,67 @@
+import DeleteIcon from '@mui/icons-material/Delete'
+import { Checkbox, FormControlLabel, Grid, IconButton, Tooltip, Typography } from '@mui/material'
+import { useGetReleasesForModelId } from 'actions/release'
+import { ChangeEvent, useCallback, useMemo, useState } from 'react'
+import Loading from 'src/common/Loading'
+import ReleaseDisplay from 'src/entry/model/releases/ReleaseDisplay'
+import MessageAlert from 'src/MessageAlert'
+import { EntryInterface, ReleaseInterface } from 'types/types'
+
+type ReleasesSelectorProps = {
+  model: EntryInterface
+  selectedReleases: ReleaseInterface[]
+  setSelectedReleases: (values: ReleaseInterface[]) => void
+}
+
+export default function ReleaseSelector({ model, selectedReleases, setSelectedReleases }: ReleasesSelectorProps) {
```

Comment:
The file name and props don't match the component name. Either make them all `ReleaseSelector` or all `ReleasesSelector`. Having inconsistent naming like this can make it a lot harder to CTRL+F to find related code.

All uses of the component should have the exact same name for the same reason, so make sure to keep `frontend/src/entry/model/mirroredModels/ExportSettings.tsx` up to date with whatever name you pick.